### PR TITLE
Re-implement NanoUI cargo console

### LINKED ISSUE #2914: Ghosts can't view the cargo shuttle console
Really minor, but when trying to interact with the cargo console as a ghost I was given 'Access Denied.' I assume this is because of the new shuttle system.

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Re-implements the NanoUI cargo console from https://github.com/ParadiseSS13/Paradise/pull/1844. All functionality should be the same: all new functionality from /tg/ (e.g. Central Command messages) has been retained.

Fixes #2914

### CODE REVIEW

**Comment by tigercat2000** [comment]
Sorry for being useless mark :crying_cat_face:

**Comment by Markolie** [comment]
No worries - it would've been easier for me to implement it anyway as I was familiar with the code already.

**Comment by Spacemanspark** [comment]
Yee haw :+1: x :100:

**Comment by TheDZD** [comment]
Does this fix #2914 ?

**Comment by Fox-McCloud** [comment]
>Only issue I can think of is that on /tg/ you can't order while the shuttle is in transit. I'm not sure why that restriction is in though, so I got rid of that to make cargo's life easier.

You may want to test out the effects of ordering when it's in transit to CentComm and when it's on its way--something tells me it may purchase the crate, only for it to end up destroyed.

File: code/controllers/Processes/shuttles/supply.dm [hunk 1]
```diff
@@ -16,9 +21,6 @@
 	height = 7
 	roundstart_move = "supply_away"
 
-/obj/docking_port/mobile/supply/New()
```
Comment (Fox-McCloud) [inline code/controllers/Processes/shuttles/supply.dm]: errr?

Comment (tigercat2000) [inline code/controllers/Processes/shuttles/supply.dm]:
```
/obj/docking_port/mobile/supply/New()
    ..()
```

it does nothing

Comment (Fox-McCloud) [inline code/controllers/Processes/shuttles/supply.dm]: true, but they're not goggles...

Comment (Markolie) [inline code/controllers/Processes/shuttles/supply.dm]: Mmmhm?

**Comment by Markolie** [comment]
It does fix #2914.

File: code/controllers/Processes/shuttles/supply.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/controllers/Processes/shuttles/supply.dm]: Was a joke, really, Mark =p

@tigercat2000  said "it does nothing". I said "they're not goggles" in reference to "the goggles do nothing!"

Comment (Markolie) [inline code/controllers/Processes/shuttles/supply.dm]: I figured as much :P.

Comment (tigercat2000) [inline code/controllers/Processes/shuttles/supply.dm]: wat

**Comment by Fox-McCloud** [comment]
@Markolie 

Just lemme know when you tested those two edge cases, if you don't mind!

**Comment by Markolie** [comment]
Good to go. Turns out that ordering while in transit means the shuttle will magically appear when the shuttle arrives.

**Comment by Fox-McCloud** [comment]
Great--will merge this in as soon as the time ticks over; this is much more convenient UI for a lot of situations.

**Pull request merged**